src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+spring.profiles.active=prod
```

Comment:
Is this prod property file or local property file?

Reply:
it is production property file